docs-src/docs/setup/install.md

```diff
@@ -35,59 +37,93 @@ This step will create an Identity & Access Management (IAM) access key from your
-<br /><br />
-<p>`l0-setup apply --dockercfg [path/to/config/file] [prefix]`</p>
-<br /><br />
-<p>If you don't have a config file yet, you can generate one by running `docker login [registry-address]`. A configuration file will be generated at `~/.docker/config.json` for Docker versions 1.7 or higher, or at `~/.dockercfg` for Docker versions below 1.7.</p>
-<br /><br />
-<p>You can modify an instance's `dockercfg` file (the one located at `~/layer0/instances/[instance]/dockercfg`) and re-run `l0-setup apply` to make changes to your authentication. Note that any EC2 instances created prior to changing your `dockercfg` file will need to be manually terminated since they only grab the `dockercfg` authentication file once during instance creation. Terminated EC2 instances will be automatically re-created by autoscaling.</p>
-</div>
+The first time you run the **apply** command, it may take around 5 minutes to complete. 
+This command is idempotent; it is safe to run multiple times if it fails the first.
 
-3. When prompted, enter the following information:
-	* **AWS Access Key ID**: The access key ID contained in the credential file that you downloaded in step 6 of the previous section.
-	* **AWS Secret Access Key**: The secret access key contained in the credential file that you downloaded in step 6 of the previous section.
-	* **Key Pair**: The name of the key pair that you created in the Prerequisites section.
-The first time you run the ```apply``` command, it may take around 15 minutes to complete. If the ```apply``` command fails to complete successfully, it is safe to run it again until it succeeds.
+It's a good idea to run the **push** command after **apply** commands complete. 
+This will send a backup of your Layer0 instance's configuration and state to S3. 
+These files can be grabbed later using the **pull** command. 
 
-## Part 4: Configure the environment variables
-Once the ```apply``` command has run successfully, you can configure the Layer0 environment variables using the ```endpoint``` command.
+!!! note "Using a Private Docker Registry"
+    **The procedures in this section are optional, but are highly recommended for production use.**
 
-To view the environment variables for your Layer0 and apply them to your shell, type the following command, replacing ```[prefix]``` with the name of the Layer0 prefix you created in Part 3:
+If you require authentication to a private Docker registry, you will need a Docker configuration file present on your machine with access to private repositories (typically located at `~/.docker/config.json`). 
 
-* (Windows PowerShell): ```l0-setup endpoint --insecure --syntax=powershell [prefix] | Out-String | Invoke-Expression```
-* (Linux/macOS): ```eval "$(l0-setup endpoint --insecure [prefix])"```
+If you don't have a config file yet, you can generate one by running `docker login [registry-address]`. 
+A configuration file will be generated at `~/.docker/config.json`.
```

Comment:
slightly confusing here to introduce that you can generate a `config.json` file w `docker` but then we create another rendered auth file `dockercfg.json`. I think if i read this without knowing the context, I would find it strange.